src/libraries/System.Net.Mail/tests/Unit/MailAddressTests/MailAddressDisplayNameTest.cs

```diff
@@ -21,6 +21,41 @@ public void MailAddress_WithUnicodeDisplayAndMailAddress_ToStringShouldReturnDis
             Assert.Equal(string.Format("\"{0}\" <{1}>", DisplayNameWithUnicode, Address), _mailAddress.ToString());
         }
 
+        [Theory]
+        [InlineData(Address, "test\"Display\"Name")]
+        [InlineData(Address, "Hello \"world hello\" world")]
+        [InlineData(Address, "Hello \"world")]
+        [InlineData(Address, "Hello \"\"world")]
+        [InlineData(Address, "\"")]
+        [InlineData(Address, "Hello \\\"world hello\\\" world")]
+        public void MailAddress_WithDoubleQuotesDisplayAndMailAddress_ToStringShouldReturnEscapedDisplayNameAndAddressInAngleBrackets(string address, string displayName)
+        {
+            MailAddress mailAddress = new MailAddress(address, displayName);
+            Assert.Equal(displayName, mailAddress.DisplayName);
+            Assert.Equal(string.Format("\"{0}\" <{1}>", displayName.Replace("\"", "\\\""), Address), mailAddress.ToString());
+        }
+
+        [Theory]
+        [InlineData(Address, "\"John Doe\"")]
+        [InlineData(Address, "\"\"")]
```

Comment:
Is it interesting to add cases like these?
```c#
[InlineData(Address, "\"\"\"")]
[InlineData(Address, "\"John \"Johnny\" Doe\"")]
```

Reply:
I think I did not understood your question correctly, can you please elaborate it ?
My assumption is you want me to add test as you shown , if it is so I can go ahead and add those test.so please suggest.

Reply:
Yes, I was suggesting to add these cases, unless you think they are useless ...

Reply:
no need for a sorry , all opinion from .net team are really valuable and I am just getting started i was not familiar so i just asked a bit of clarification  , as you suggested i have added the test case for both values , looking forward for learning more and contributing more ,  .net is really awesome and getting better and better every day ❤ for  .net and thanks for all kind of help  @ManickaP  @karelz  @wfurt